pytests/tests/test_signature.py

```diff
@@ -0,0 +1,90 @@
+#
+# Copyright (C) 2019 - 2020 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the GNU General Public License v2 (the "License");
+# you may not use this file except in compliance with the License. The terms
+# of the License are located in the COPYING file of this distribution.
+#
+#   Author: Siddharth Chandrasekaran <csiddharth@vmware.com>
```

Comment:
Please add your name or remove this line entirely.

Reply:
Done.